tests/plugins/common/initialize_test.py

```diff
@@ -10,37 +10,45 @@ class TestFromPluginClassname:
 
     def test_success(self):
         plugin = initialize.from_plugin_classname(
-            'HexHighEntropyString',
+            plugin_classname='HexHighEntropyString',
+            custom_plugin_paths=[],
             hex_limit=4,
         )
 
-        assert isinstance(plugin, HexHighEntropyString)
+        # Dynamically imported classes have different
+        # addresses for the same functions as statically
+        # imported classes do, so isinstance does not work.
+        assert str(plugin.__class__) == str(HexHighEntropyString)
+        assert dir(plugin.__class__) == dir(HexHighEntropyString)
```

Comment:
Had to do
```
-        assert all(
-            func(plugin.__class__) == func(Base64HighEntropyString)
-            for func in (str, dir)
-        )
+        assert str(plugin.__class__) == str(Base64HighEntropyString)
+        assert dir(plugin.__class__) == dir(Base64HighEntropyString)
```
because otherwise it was `94%   21->exit, 71->exit`